website/templates/profile/addons.mako

```diff
@@ -102,4 +99,17 @@
     % for js_asset in addon_js:
       <script src="${js_asset}"></script>
     % endfor
+
+   <script src="/static/public/js/addon-permissions.js"></script>
+
+   <script type="text/javascript">
+        for (var i=0; i < ${len(addon_enabled_settings)}; i++) {
+           var addonName = ${addon_enabled_settings}[i];
```

Comment:
To be safe, we should use `json.dumps(addon_enabled_settings)`, just in case any of the strings in the list are not JSON-compatible. This can happen with unicode strings, which will get the `u` prefix on Python 2.